Shared/Realm.Sync.Shared/User.cs

```diff
@@ -88,11 +93,29 @@ public static User[] AllLoggedIn
             return new User(SyncUserHandle.GetSyncUser(refresh_token["token_data"]["identity"], refresh_token["token"], serverUrl.AbsoluteUri, false));
         }
 
-        static User()
+        /// <summary>
+        /// Configures user persistence.
```

Comment:
We should mention that this should be called before `User.Current`, `User.All`, or `User.LoginAsync`.